load-tests/src/db-lockup-test.ts

```diff
@@ -0,0 +1,82 @@
+import { sleep } from "k6";
+import { req } from './requests';
+import type { User } from "./types/auth";
+import {CreatePlanInput} from "./types/plan";
+import {ActivityInsertInput} from "./types/activity";
+
+export const options = {
+  // A number specifying the number of VUs to run concurrently.
+  vus: 50,
+  // A string specifying the total duration of the test run.
+  duration: '10s',
+
+  // If the statistics of the load test fall below the following threshold, it with exit with a nonzero exit code
+  thresholds: {
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(50)<300', 'p(95)<1000'], // 50% below 300ms, 95% of requests should be below 1000ms
```

Comment:
not a huge deal since this is going to need tuning based on compute resources (e.g. running these in GH actions), but this `http_req_duration` threshold actually fails for me locally.
<img width="1090" alt="Screenshot 2024-01-09 at 4 00 54 PM" src="https://github.com/NASA-AMMOS/aerie/assets/49844593/ef8a8ba0-d59d-42b5-92b6-688b6090c30c">

Reply:
Yeah, I've noticed that that threshold is flaky. I think setting it higher in this file might make sense (since duping a large plan is a large transaction), but I'm not sure what a good value is.